static_src/util/create_styler.js

```diff
@@ -0,0 +1,17 @@
+import classNames from 'classnames';
+
+function uniqueOnly(value, index, self) {
+  return self.indexOf(value) === index;
+}
+
+export default function createStyler(...args) {
```

Comment:
Ya, I don't think that would happen. I think the hashing on the end of CSS module classes is there to prevent that